commit ec39bbfd55d07de2e2d4111f35c7ad9523c89ec3
Author: Oliver Upton <oliver.upton@linux.dev>
Date:   Mon Apr 22 20:01:50 2024 +0000

    KVM: arm64: vgic-its: Rip out the global translation cache
    
    The MSI injection fast path has been transitioned away from the global
    translation cache. Rip it out.
    
    Signed-off-by: Oliver Upton <oliver.upton@linux.dev>
    Link: https://lore.kernel.org/r/20240422200158.2606761-12-oliver.upton@linux.dev
    Signed-off-by: Marc Zyngier <maz@kernel.org>

diff --git a/include/kvm/arm_vgic.h b/include/kvm/arm_vgic.h
index c15e7fcccb86..76ed097500c0 100644
--- a/include/kvm/arm_vgic.h
+++ b/include/kvm/arm_vgic.h
@@ -286,9 +286,6 @@ struct vgic_dist {
 #define LPI_XA_MARK_DEBUG_ITER	XA_MARK_0
 	struct xarray		lpi_xa;
 
-	/* LPI translation cache */
-	struct list_head	lpi_translation_cache;
-
 	/* used by vgic-debug */
 	struct vgic_state_iter *iter;